src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -151,7 +151,7 @@ struct H5WorldtubeBoundaryDataManager : db::SimpleTag {
   }
 };
 
-struct LMax : db::SimpleTag {
+struct LMax : Spectral::Swsh::Tags::LMax {
```

Comment:
Yeah, this inheritance lets me use this tag as the single tag in the const global cache for passing to both the Cce utilities and the Swsh utilities. Depending on the comment intent, my responses are:
- remove the initialization tag : Then I'm not sure how to get it in the const global cache from the option values
- remove the simple tag : Then it requires some option infrastructure brought into the Swsh namespace to support the tag there, which is fine but the Swsh library wrappers are currently independent of options and option construction.
- call this something other than an `InitializationTag` because it goes in the const global cache : sure, that's probably clearer

Reply:
My recommendation is to just specify the simple tag in the `const_global_cache_tags` type alias and add the `create_from_options` and `option_tags` to the existing simple tags

Reply:
That's a little bit of an awkward design, though, due to the nature of the `LMax` option tag - it's in the Cce group, and it seems undesirable to have the general swsh tag have the behavior of retrieving information from the Cce option group. I could make it a `Swsh` option group, but then a given executable wouldn't be able to separately specify an `LMax` resolution for Cce and for other spin-weighted computations. The hope with inheritance was that it ensures that the Cce databoxes will use the Cce `LMax` appropriately with the swsh utilities, but still keep other components free to specify their own `LMax` when interacting with the spin-weighted utilities.